src/ConfidentialLedger/Az.ConfidentialLedger.psd1

```diff
@@ -1,7 +1,7 @@
 @{
   GUID = 'c7eaa9c1-72c4-4251-9e7d-bf9b8e8fa71e'
   RootModule = './Az.ConfidentialLedger.psm1'
-  ModuleVersion = '0.1.0'
```

Comment:
This change is not required. The version will be handled by us after code is merged into main branch.